### PR TITLE
Fixed PUT method in Field/{id} resource (Feature/#175)

### LINKED ISSUE #175: Fix PUT method in Field/{id} resource
- fix response body;
- fix problem with range updating functionality;

### PR DESCRIPTION
**Pull request by zzeropilot**

### CODE REVIEW

File: src/app/schemas/field.py [hunk 1]
```diff
@@ -123,15 +123,27 @@ class Meta:
...
-    range = fields.Nested(RangeSchema, required=False)
-    added_choice_options = fields.List(fields.Str(), required=False)
-    removed_choice_options = fields.List(fields.Str(), required=False)
-    updated_autocomplete = fields.Nested(SettingAutocompleteSchema, required=False)
+    updated_name = fields.Str(required=False, data_key="updatedName")
+    range = fields.Nested(RangeSchema)
+    added_choice_options = fields.List(fields.Str(),
+                                       required=False,
+                                       data_key="addedChoiceOptions"
+                                       )
```
Comment (yurdosii) [inline src/app/schemas/field.py]: Formatting

File: src/app/services/field.py [hunk 2]
```diff
@@ -637,23 +649,31 @@ def update_radio_field(
...
-                option = ChoiceOptionService.get_by_field_and_text(field_id=field_id,
-                                                                   option_text=removed_option)
-                option_update = ChoiceOptionService.delete(option_id=option.id)
-                if option_update is None:
+                radio_option = ChoiceOptionService.get_by_field_and_text(
+                    field_id=field.id,
+                    option_text=removed_option
+                )
+                radio_option_update = ChoiceOptionService.delete(option_id=radio_option.id)
+                if radio_option_update is None:
```
Comment (AndriiKret) [inline src/app/services/field.py]: Add checker if radio_option exist

Comment (zzeropilot) [reply src/app/services/field.py]: fixed

File: src/app/schemas/field.py (hunk 1 above)
Comment (zzeropilot) [reply src/app/schemas/field.py]: fixed

**Review by yurdosii — DISMISSED** [review]

**Review by yurdosii — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Pull request merged**